ntc_templates/templates/index

```diff
@@ -491,6 +491,8 @@ ruckus_fastiron_show_mac-address.textfsm, .*, ruckus_fastiron, sh[[ow]] mac[[-ad
 ruckus_fastiron_show_version.textfsm, .*, ruckus_fastiron, sh[[ow]] v[[ersion]]
 ruckus_fastiron_show_arp.textfsm, .*, ruckus_fastiron, sh[[ow]] ar[[p]]
 
+ubiquiti_edgerouter_show_interfaces_ethernet_physical.textfsm, .*, ubiquiti_edgerouter, sh[[ow]] interfaces ethernet (\S+\s)?physical
```

Comment:
Does the command " show interfaces ethernet" work? I am not in favor of adding more templates if the details can be captured in other ways.

Reply:
Sorry, it took us a while to get back in lab an ubiquiti to test. And then I forgot to update...
We are missing a few information without the "physical":

```
admin@staff:~$ show interfaces ethernet 
Codes: S - State, L - Link, u - Up, D - Down, A - Admin Down
Interface    IP Address                        S/L  Description                 
---------    ----------                        ---  -----------                 
eth0         -                                 u/D  Port                        
eth1         -                                 u/D  Port                        
eth2         -                                 u/D  Port                        
eth3         -                                 u/D  Port                        
eth4         -                                 u/u  IP-278903-87981             
admin@staff:~$ show interfaces ethernet physical 
Settings for eth0:
   Auto-negotiation: on
   Speed: 10Mb/s
   Duplex: Half
   Link detected: no

Settings for eth1:
   Auto-negotiation: on
   Speed: 10Mb/s
   Duplex: Half
   Link detected: no

Settings for eth2:
   Auto-negotiation: on
   Speed: 10Mb/s
```

On our side we are particularly interested by the speed (also sometimes use duplex and auto-negotiation info).